Updating maintenance page and removing reference to error

phutchins opened the pull request.

phutchins committed:
Updating maintenance page and removing reference to error

metafraction reviewed (approved):
<img width="1191" alt="Screen Shot 2022-01-19 at 2 50 06 PM" src="https://user-images.githubusercontent.com/84931414/150211631-703400e3-890a-4194-8936-7a27320955be.png">

looking good!

phutchins committed:
Publish

 - @dgrants/app@0.2.43-b96899d.0
 - @dgrants/contracts@0.2.31-b96899d.0
 - @dgrants/dcurve@0.2.33-b96899d.0
 - @dgrants/types@0.2.31-b96899d.0
